_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,321 @@
+  using Ledger’s [explorers][topic block explorers].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is the difference between "native segwit" and "bech32"?]({{bse}}100434)
+  Murch describes that bech32 is the address format for encoding native segwit
+  v0 outputs and goes on to explain that due to a [length extension mutation][news78 bech32 mutability],
+  a modified bech32 address format, "bech32m", will encode Pay-to-Taproot (P2TR)
+  segwit v1 outputs.
+
+- [What makes cross input signature aggregation complicated to implement?]({{bse}}100216)
+  Michael Folkson references a Bitcoin dev mailing list [post by AJ Towns][aj signature aggregation]
+  in explaining challenges to implementing cross input signature aggregation in Bitcoin.
+
+- [How do BIP8 and BIP9 differ, how are they alike?]({{bse}}100490)
+  Murch provides an overview of early soft fork activation methods using the
+  block version, BIP9's multiple-proposal-supporting activation method, and BIP8's block
+  height and lock-in adjustments to BIP9.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update
+
+- [Bitcoin Core 0.21.0rc3][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20564][] makes two changes to the way that Bitcoin Core
+  signals support for `addrv2` messages ([BIP155][]):
+
+  - Bitcoin Core will only negotiate support for `addrv2` messages with peers that signal
+    they are using p2p version 70016 or higher. This restriction isn't required by the BIP,
+    but release testing has revealed that some other implementation will disconnect
+    from Bitcoin Core if they receive any unknown message, including
+    `sendaddrv2`. This change may be reverted in future versions of Bitcoin Core,
+    so other implementations are advised to always tolerate unknown p2p messages.
```

Comment:
Harding has resolved this as "are advised to tolerate unknown P2P messages at any time in the connection" which is clearer. In general, there's no problem in splitting infinitives, and the alternative offered here ("always advised to tolerate") changes the meaning of the sentence (the advice is happening always instead of the toleration).